tests/components/inels/conftest.py

```diff
@@ -0,0 +1,125 @@
+"""Test fixtures."""
+
+from typing import Any
+from unittest.mock import AsyncMock, Mock, patch
+
+import pytest
+
+from homeassistant.core import HomeAssistant
+
+from . import HA_INELS_PATH
+from .common import DOMAIN, MockConfigEntry, get_entity, set_mock_mqtt
+
+
+@pytest.fixture(autouse=True)
+def auto_enable_custom_integrations(enable_custom_integrations: None):
+    """Enable custom integrations for testing."""
+    return
```

Comment:
Do you need this when not testing with iNELS as a custom integration? If not, please remove it.